omod/src/main/webapp/resources/scripts/diagnoses/diagnoses.js

```diff
@@ -1,10 +1,17 @@
 (function(diagnoses, $, _, undefined) {
-
+    
     var codingSystemToUse = 'ICD-10-WHO';
 
     var mapTypeOrder = [ "SAME-AS", "NARROWER-THAN" ]
+    
+    findPreferredCodingSource = function(preferredCodingSource) {
```

Comment:
does this really need to take in a "preferreedConfingSource"... it makes it confusing... I whould thing this function shouldn't need to tak ein a parameter...